usr.bin/du/du.c

```diff
@@ -273,15 +280,17 @@ main(int argc, char *argv[])
 			if (p->fts_level <= depth && threshold <=
 			    threshold_sign * howmany(p->fts_bignum *
 			    cblocksize, blocksize)) {
+				xo_open_instance("paths");
```

Comment:
unless I'm missing something, and maybe I am, why are we opening this twice? Here and at line 263?